### PR TITLE
NFT-327 feat: add TOS to wallet connection modal

### PR DESCRIPTION
**Pull request by cnasc**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "@hookform/resolvers": "2.8.8",
     "@pinata/ipfs-gateway-tools": "1.1.6",
     "@prisma/client": "3.11.0",
-    "@rainbow-me/rainbowkit": "0.2.0",
+    "@rainbow-me/rainbowkit": "0.2.1",
```
Comment (cnasc) [inline package.json:34]: had to update for the `DisclaimerComponent` to appear

File: components/ApplicationProviders/ApplicationProviders.tsx [hunk 2]
```diff
@@ -27,6 +28,15 @@ const CHAINS: Chain[] = [
   { ...chain.polygon, name: 'Polygon' },
 ];
 
+const Disclaimer: DisclaimerComponent = ({ Text, Link }) => (
+  <Text>
+    By connecting your wallet, you agree to the{' '}
```
Comment (cnasc) [inline components/ApplicationProviders/ApplicationProviders.tsx:33]: paging Dr. @jringenberg for wording

Comment (jringenberg) [reply components/ApplicationProviders/ApplicationProviders.tsx:33]: A very solid boilerplate! I like it. Let's say "Backed" in there specifically so it doesn't seem like rainbowkit's ToS, maybe  slip it in like "...you agree to the **Backed Terms of Service** and acknowledge you have read and understand Backed's **Privacy Policy**."

Comment (cnasc) [reply components/ApplicationProviders/ApplicationProviders.tsx:33]: Updated, also took privacy policy out in favor of protocol disclaimer

**Review by adamgobes — APPROVED** [review]

**Pull request merged**